packages/cfd/src/features/Components/proof-of-identity.tsx

```diff
@@ -0,0 +1,49 @@
+// @ts-expect-error remove this line when ProofOfIdentityContainerForMt5 is converted to TS
+import ProofOfIdentityContainerForMt5 from '@deriv/account/src/Sections/Verification/ProofOfIdentity/proof-of-identity-container-for-mt5.jsx';
+import React from 'react';
+import { useStore, observer } from '@deriv/stores';
+import type { TCoreStores } from '@deriv/stores/types';
+
+type TCFDValue = {
+    poi_state: string;
+};
+
+type TFormValues = {
+    poi_state?: string;
+};
+
+export type TProofOfIdentity = {
+    index: number;
+    onSubmit: (index: number, value: TCFDValue) => void;
+    value: TCFDValue;
+    addNotificationMessageByKey: TCoreStores['notifications']['addNotificationMessageByKey'];
+    height: string;
+    onSave: (index: number, values: TFormValues) => void;
+    removeNotificationByKey: TCoreStores['notifications']['removeNotificationByKey'];
+    removeNotificationMessage: TCoreStores['notifications']['removeNotificationMessage'];
+    jurisdiction_selected_shortcode: string;
+};
+
+const ProofOfIdentity = observer(({ index, onSave, onSubmit, ...props }: TProofOfIdentity) => {
```

Comment:
Pls avoid using ...props. When you use ...props, it's not clear which props are being passed down. TypeScript can't check that the props you're passing are the right types if it doesn't know what props are being passed. Spreading props can unintentionally override other props